integrations/cdi/jpa-cdi/pom.xml

```diff
@@ -274,7 +270,7 @@
                 <dependency>
                     <groupId>javax.activation</groupId>
                     <artifactId>javax.activation-api</artifactId>
-                    <scope>runtime</scope>
+                    <scope>compile</scope>
```

Comment:
Is the scope change needed ?

Reply:
java.xml.bind requires java.activation, so java.activation needs to be on the module-path. To get the compiler plugin to do that I needed to change to compile scope.  I have updated the jpa-cdi module-info and removed the requires java.activation since this module (jpa-cdi) does not require java.activation.